packages/connect-web/package.json

```
@@ -10,9 +10,14 @@
   "scripts": {
     "clean": "rm -rf ./dist/*",
     "build": "npm run build:cjs && npm run build:esm",
-    "build:cjs": "tsc --project tsconfig.json --module commonjs --outDir ./dist/cjs --declaration --declarationDir ./dist/cjs && echo >./dist/cjs/package.json '{\"type\":\"commonjs\"}'",
-    "build:esm": "tsc --project tsconfig.json --module ES2015 --verbatimModuleSyntax --outDir ./dist/esm --declaration --declarationDir ./dist/esm",
-    "attw": "attw --pack"
+    "build:cjs": "tsc --project tsconfig.build.json --module commonjs --outDir ./dist/cjs --declaration --declarationDir ./dist/cjs && echo >./dist/cjs/package.json '{\"type\":\"commonjs\"}'",
+    "build:esm": "tsc --project tsconfig.build.json --module ES2015 --verbatimModuleSyntax --outDir ./dist/esm --declaration --declarationDir ./dist/esm",
+    "attw": "attw --pack",
+    "conformance:client:chrome": "./conformance/bin/connectconformance --mode client --conf ./conformance/conformance-web.yaml -v -- ./conformance/bin/conformanceclient --browser chrome --headless",
```

The package `@connectrpc/connect-conformance` should have a "binary" (package.json `bin` field) for the conformance test runner `connectconformance`. 

This will make the binary available everywhere, and we don't need multiple copies of the file `conformance/bin/connectconformance` everywhere.

The call here can then be:

```suggestion
    "conformance:client:chrome": "connectconformance --mode client --conf ./conformance/conformance-web.yaml -v -- ./conformance/bin/conformanceclient --browser chrome --headless",
```

Same for all other packages using the conformance test runner.

---

Ah, sweet. Yeah, thanks for that. Updated.